proto/xudrpc.proto

```diff
@@ -527,22 +527,26 @@ message SwapResult {
   double quantity = 4 [json_name = "quantity"];
   // The hex-encoded payment hash for the swaps.
   string r_hash = 5 [json_name = "r_hash"];
+  // The hex-encoded preimage.
+  string r_preimage = 6 [json_name = "r_preimage"];
+  // The price of the order that's being executed.
```

Comment:
I'd say here "The price used for the swap" or something like that which indicates that the swap has already happened.